tox.ini

```diff
@@ -25,6 +25,7 @@ exclude =
     venv/
     sagemaker-tensorflow-extensions
     benchmarks/
+    scripts/
```

Comment:
why exclude these? there are Python files written by us in that directory

Reply:
will add